test/jop_relumin.jl

```diff
@@ -0,0 +1,40 @@
+using LinearAlgebra, Jets, JetPack, Test
+
+# note no need for a linearity test as function is "linear" for x > 0 and constant for x <= 0 
```

Comment:
Is this comment in error.  It seems to contradict the fact that there is a linearity test below.